### PR TITLE
LPS 101618 Deprecate 'Liferay.Util.toggleDisabled'

### PR DESCRIPTION
**Pull request by mateomustapic**
This is the new VanillaJS implementation of toggleDisabled utility.

### CODE REVIEW

**Comment by mateomustapic** [comment]
ci:test:sf

**Comment by mateomustapic** [comment]
ci:test:stable

**Comment by mateomustapic** [comment]
ci:test:relevant

**Comment by julien** [comment]
Hey @mateomustapic just one little thing. 

I think we would have been able to put this in a single commit with a "description" (that I can't see in any of these commits).  It might not seem important (and of course and it doesn't mean that the code changes are neither good nor bad) but I think it actually adds a lot of value and [is](https://github.com/liferay/liferay-frontend-guidelines/blob/master/general/pull_requests.md) actually important.

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/toggle_disabled.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		nodes = [nodes._node];
+	}
+	else if (nodes._nodes) {
+		nodes = nodes._nodes;
+	}
+	else if (nodes.nodeType === 1) {
+		nodes = [nodes];
+	}
+	nodes.forEach((node) => {
+		node.disabled = state;
```
Comment (julien) [inline modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/util/toggle_disabled.js:34]: I know this was actually the case before this change, but we could probably improve this.

If someone calls this function like this 

```js
toggleDisabled('a', 'whatever i want to put here')
```

We might get strange results.

I was reading [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled) article about the `disabled` attribute and it seems that we should accept a boolean.

**Comment by julien** [comment]
ci:stop

**Comment by julien** [comment]
send [here](#273)

**Comment by julien** [comment]
ci:close

**Pull request closed without merging**